sass/base/_reset.scss

```diff
@@ -0,0 +1,25 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
What's interesting is that I had originally included the box-sizing in our reset as outlined in the article, but for some reason the hover overlay in the More Apps section wasn't inheriting the box-sizing properly until I switched it to what we currently have.  Thoughts?